src/olympia/addons/cron.py

```diff
@@ -278,14 +278,14 @@ def reindex_addons(index=None, addon_type=None):
 
 def cleanup_image_files():
     """
-    Clean up all header and footer images files for themes.
+    Clean up all header images files for themes.
 
     We use these images to asynchronuously generate thumbnails with
     tasks, here we delete images that are older than one day.
 
     """
     log.info('Removing one day old temporary image files for themes.')
-    for folder in ('persona_footer', 'persona_header'):
+    for folder in ('persona_header'):
```

Comment:
This is a bit of a problem, `('persona_header')` is actually equivalent with the string `persona_header` so that if you do the for-loop here it'll iterate over `p`, `e`, `r`, `s`, `o`, `n`, `a`, `_`, `h`, etc... instead of that folder. It's probably best to either remove the for-loop or to add an ending comma to the tuple like this: `('persona_header', )`